sentry_streams/sentry_streams/adapters/arroyo/rust_step.py

```diff
@@ -322,15 +324,19 @@ def __yield_messages(self) -> MutableSequence[Tuple[PipelineMessage, Committable
         for message, committable in self.__retriever.fetch():
             ret.append((message, committable))
             self.__globbed_committable.update(committable)
+            logger.info("__yield_messages: globbed_committable is", self.__globbed_committable)
             watermarks = self.__watermarks.copy()
+            logger.info(f"_yield_messages: watermarks is {watermarks}")
```

Comment:
I assume these are not going to be merged. We cannot log at INFO level for every message.